cg/apps/lims/api.py

```diff
@@ -395,30 +396,38 @@ def get_sample_project(self, sample_id: str) -> str:
         """Get the lims-id for the project of the sample"""
         return self.sample(sample_id).get("project").get("id")
 
-    def get_sample_rin(self, sample_id: str) -> float:
+    def get_sample_rin(self, sample_id: str) -> float | None:
         """Return the sample RIN value."""
-        sample_artifact: Artifact = Artifact(self, id=f"{sample_id}PA1")
-        return sample_artifact.udf.get(PROP2UDF["rin"])
+        rin = None
+        try:
+            sample_artifact: Artifact = Artifact(self, id=f"{sample_id}PA1")
```

Comment:
```suggestion
            sample_artifact = Artifact(self, id=f"{sample_id}PA1")
```

Reply:
@Karl-Svard What is the `PA1` constant? Can we make it a named variable?

Reply:
Added it to the issue for future refactoring: https://github.com/Clinical-Genomics/cg/issues/3163